packages/web/src/common/store/pages/audio-rewards/sagas.ts

```diff
@@ -256,11 +256,13 @@ function* claimChallengeRewardAsync(
   }
   let aaoErrorCode
   try {
-    const challenges = specifiers.map(({ specifier, amount }) => ({
-      challenge_id: challengeId,
-      specifier,
-      amount
-    }))
+    const challenges = specifiers
```

Comment:
What about mobile? Wondering if this should be done at the saga layer where we try to do the claims?

Reply:
i believe this is the saga layer, it's just in web/common but it gets imported to mobile at the end of the day. this is the in-flight common migration.

Reply:
Oh duh, you're right. I stopped reading after `packages/web`. Carry on!